diff -Nur netbeans-6.9/libs.svnClientAdapter/src/org/netbeans/libs/svnclientadapter/SvnClientAdapterFactory.java netbeans-6.9.new/libs.svnClientAdapter/src/org/netbeans/libs/svnclientadapter/SvnClientAdapterFactory.java
--- netbeans-6.9/libs.svnClientAdapter/src/org/netbeans/libs/svnclientadapter/SvnClientAdapterFactory.java	2010-06-10 15:20:26.000000000 +0400
+++ netbeans-6.9.new/libs.svnClientAdapter/src/org/netbeans/libs/svnclientadapter/SvnClientAdapterFactory.java	2010-07-20 06:58:22.586364869 +0400
@@ -48,7 +48,7 @@
 import org.tigris.subversion.svnclientadapter.SVNClientException;
 import org.tigris.subversion.svnclientadapter.javahl.JhlClientAdapter;
 import org.tigris.subversion.svnclientadapter.javahl.JhlClientAdapterFactory;
-import org.tigris.subversion.svnclientadapter.svnkit.SvnKitClientAdapterFactory;
+//import org.tigris.subversion.svnclientadapter.svnkit.SvnKitClientAdapterFactory;
 
 /**
  *
@@ -101,8 +101,9 @@
                 return JhlClientAdapterFactory.isAvailable();
             }
             case svnkit: {
-                SvnKitClientAdapterFactory.setup();
-                return SvnKitClientAdapterFactory.isAvailable();
+//                SvnKitClientAdapterFactory.setup();
+//                return SvnKitClientAdapterFactory.isAvailable();
+                return false;
             }
         }
         return false;
@@ -114,7 +115,8 @@
                 return JhlClientAdapterFactory.createSVNClient(JhlClientAdapterFactory.JAVAHL_CLIENT);
             }
             case svnkit: {
-                return SvnKitClientAdapterFactory.createSVNClient(SvnKitClientAdapterFactory.SVNKIT_CLIENT); 
+//                return SvnKitClientAdapterFactory.createSVNClient(SvnKitClientAdapterFactory.SVNKIT_CLIENT);
+                return null;
             }
         }
         return null;
